### DISCUSSION TITLE
ARK: Ascended Support?

### DISCUSSION THREAD

**Discussion by mcbaer** [coldino/Larkator#91] [Ideas]
Is there any intention of updating this tool to work with A:SE dlc "ARK: Survival Ascended"?

Personally prefer not having to install mods into my private dedicated server; and this was the only tool with the feature set available for A:SE...

**Comment by coldino** [comment]
I'd like to, but this isn't possible yet. The save file is a new and very different format so it'll take time to develop a method of reading it.

**Reply by mcbaer** [reply]
Might have a look here https://github.com/Kakoen/ark-sa-save-tools/tree/master .. if you haven't already..

**Reply by coldino** [reply]
Oh thanks, this one was not on my radar yet. Looks like it needs a bit of time to mature, and it's for Java not the .NET infrastructure so will take some effort to port over. Still, a good find.

**Reply by mcbaer** [reply]
np ;) 

Or could move Larkator over to the java side; much better UI control there - less dependencies on Micro(Sh*t)soft .. unless .NET is your "bread & butter.." for coding.

Either way, hope you're able to get Larkator working with ASA soon.. annoyed with spending the couple hours a day available to play, flying around looking for those Dinos with level worth taming.. lol
